src/site/_includes/layout.njk

```diff
@@ -38,6 +38,7 @@
       snackbar will be the first stop for the screen reader / focus.
     #}
     <web-snackbar-container></web-snackbar-container>
+    <div class="w-loading-progress"></div>
```

Comment:
Would using `<progress>` make life difficult? Was just thinking it might be nice semantically. But if it's fiddly, don't worry about it.

If we're going with a div, I'd suggest adding `role=progressbar aria-valuemin=0 aria-valuemax=100`. By not providing an `aria-valuenow` attribute, it conveys that it's indeterminate. [Explainer](https://www.w3.org/TR/wai-aria-1.1/#progressbar).

Reply:
I tried `<progress>` but it was difficult to style it correctly (because of `-webkit-appearance` and friends). I did the second approach.